crates/fee/src/lib.rs

```diff
@@ -387,6 +540,60 @@ impl<T: Config> Pallet<T> {
         Ok(())
     }
 
+    pub fn _set_issue_fee(
+        fee: UnsignedFixedPoint<T>,
+    ) {
+        IssueFee::<T>::put(fee);
+    }
+
+    pub fn _set_issue_griefing_collateral(
+        griefing_collateral: UnsignedFixedPoint<T>,
+    ) {
+        IssueGriefingCollateral::<T>::put(griefing_collateral);
+    }
+
+    pub fn _set_redeem_fee(
+        fee: UnsignedFixedPoint<T>,
+    ) {
+        RedeemFee::<T>::put(fee);
+    }
+
+    pub fn _set_refund_fee(
+        fee: UnsignedFixedPoint<T>,
+    ) {
+        RefundFee::<T>::put(fee);
+    }
+
+    pub fn _set_premium_redeem_fee(
+        fee: UnsignedFixedPoint<T>,
+    ) {
+        PremiumRedeemFee::<T>::put(fee);
+    }
+
+    pub fn _set_punishment_fee(
+        fee: UnsignedFixedPoint<T>,
+    ) {
+        PunishmentFee::<T>::put(fee);
+    }
+
+    pub fn _set_replace_griefing_collateral(
+        griefing_collateral: UnsignedFixedPoint<T>,
+    ) {
+        ReplaceGriefingCollateral::<T>::put(griefing_collateral);
+    }
+
+    pub fn _set_theft_fee(
+        fee: UnsignedFixedPoint<T>,
+    ) {
+        TheftFee::<T>::put(fee);
```

Comment:
Should we put some input validation on these functions? Having a value of > 100% does not make sense. Then again, this is only called by root so maybe we don't need to? @gregdhill what do you think?

Reply:
I think it does make sense to do some basic validation since these setters are specifically meant to make the UX easier for end-users.